processor.py

```diff
@@ -189,13 +189,13 @@ def init():
             A = getmatrix()
             print("The result is: ")
             trans_dict = {
-                1: A.transpose(),
-                2: A.side_transpose(),
-                3: A.ver_transpose(),
-                4: A.hori_transpose()
+                1: A.transpose,
+                2: A.side_transpose,
+                3: A.ver_transpose,
+                4: A.hori_transpose
             }
```

Comment:
This is a good observation. This should significantly improve speed since all the options are not being evaluated at runtime, but only on request. This is a good change, and should be implemented.